src/server_manager/web_app/cloud_accounts.ts

```diff
@@ -14,17 +14,34 @@
 
 import {Account} from '../model/digitalocean';
 
-type DigitalOceanAccountFactory = (accessToken: string) => Account;
+type DigitalOceanAccountFactory =
+    (id: string, name: string, accessToken: string) => Account;
+
+type PersistedDigitalOceanAccount = {
+  id: string,
+  name: string,
+  credential: string,
+};
+
+enum CloudId {
+  DigitalOcean,
+}
 
 export class CloudAccounts {
+  // TODO: We need to migrate this if we decide to use the same key.
   private readonly DIGITALOCEAN_TOKEN_STORAGE_KEY = 'LastDOToken';
```

Comment:
If we are changing the format, there's no point in preserving the same storage key. You're breaking backward-compatibility anyway.

We should look forward the think about a format that works for multiple clouds and multiple accounts.
I think a list would be better, rather than a key per cloud. We can ensure uniqueness per cloud in other ways, but let's not constrain serialization.

Reply:
Yep, you're right. I'm slowly migrating to a separate storage key. I just added that in this latest commit.

I haven't tackled the backwards compatibility yet and will do that in the next commit.